okhttp/src/main/java/okhttp3/ConnectionPool.java

```diff
@@ -110,17 +110,19 @@ public synchronized int connectionCount() {
   }
 
   /**
-   * Acquires a recycled connection to {@code address} for {@code streamAllocation}. If non-null
-   * {@code route} is the resolved route for a connection.
+   * Acquires a recycled connection to {@code address} for {@code transmitter}. If non-null {@code
+   * route} is the resolved route for a connection.
```

Comment:
Do you usually specify the returned parameter in comments as well? Like _returns true if was able to acquire a connection_ kind of thing

Reply:
Yes, definitely. Lemme fix!